pyrdp/ui/qt.py

```diff
@@ -194,6 +195,12 @@ def scale(self, scale):
         :param scale: Ex: 0.5 for 50% height and 50% width.
         """
         self.ratio = scale
+        if not self.scaleToWindow:
+            # resize widget to session size
+            super().resize(self.sessionWidth, self.sessionHeight)
```

Comment:
wouldn't it be more readable using `if condition x else y`  instead of `if not condition y else x`?

Reply:
Good point

Reply:
fixed